src/content/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-tutorial-advanced-dashboarding.mdx

```diff
@@ -0,0 +1,471 @@
+---
+title: "Advance your dashboarding with NRQL"
+tags:
+  - NRQL tutorial
+  - 'NRQL: New Relic query language'
+  - Get started
+  - Tutorial
+metaDescription: "Learn how to use NRQL to improve your dashboarding"
+---
+
+import queriesnrql2tutorial1 from 'images/queries-nrql_screenshot-crop-tutorial2-1.webp'
+
+import queriesnrql2tutorial2 from 'images/queries-nrql_screenshot-crop-tutorial2-2.webp'
+
+import queriesnrql2tutorial3 from 'images/queries-nrql_screenshot-crop-tutorial2-3.webp'
+
+import queriesnrql2tutorial4 from 'images/queries-nrql_screenshot-crop-tutorial2-4.webp'
+
+import queriesnrql2tutorial5 from 'images/queries-nrql_screenshot-crop-tutorial2-5.webp'
+
+import queriesnrql2tutorial6 from 'images/queries-nrql_screenshot-crop-tutorial2-6.webp'
+
+import queriesnrql2tutorial7 from 'images/queries-nrql_screenshot-crop-tutorial2-7.webp'
+
+import queriesnrql2tutorial8 from 'images/queries-nrql_screenshot-crop-tutorial2-8.webp'
+
+import queriesnrql2tutorial9 from 'images/queries-nrql_screenshot-crop-tutorial2-9.webp'
+
+import queriesnrql2tutorial10 from 'images/queries-nrql_screenshot-crop-tutorial2-10.webp'
+
+import queriesnrql2tutorial11 from 'images/queries-nrql_screenshot-crop-tutorial2-11.webp'
+
+import queriesnrql2tutorial12 from 'images/queries-nrql_screenshot-crop-tutorial2-12.webp'
+
+import queriesnrql2tutorial13 from 'images/queries-nrql_screenshot-crop-tutorial2-13.webp'
+
+import queriesnrql2tutorial14 from 'images/queries-nrql_screenshot-crop-tutorial2-14.webp'
+
+import queriesnrql2tutorial15 from 'images/queries-nrql_screenshot-crop-tutorial2-15.webp'
+
+import queriesnrql2tutorial16 from 'images/queries-nrql_screenshot-crop-tutorial2-16.webp'
+
+import queriesnrql2tutorial17 from 'images/queries-nrql_screenshot-crop-tutorial2-17.webp'
+
+import queriesnrql2tutorial18 from 'images/queries-nrql_screenshot-crop-tutorial2-18.webp'
+
+import queriesnrql2tutorial19 from 'images/queries-nrql_screenshot-crop-tutorial2-19.webp'
+
+import queriesnrql2tutorial20 from 'images/queries-nrql_screenshot-crop-tutorial2-20.webp'
+
+We hope seeing example queries and explanations in the context of your own data helps you better understand how to transform data into powerful visuals. In this tutorial, you'll use what you learned in our previous tutorials on the [foundations of using NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial/) and [controlling your data with NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-tutorial-control-your-data/) and build on them with more advanced concepts.
```

Comment:
```suggestion
We hope seeing example queries and explanations in the context of your own data helps you better understand how to transform data into powerful visuals. In this tutorial, you'll build on concepts learned in [foundations of using NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-tutorial/) and [controlling your data with NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/nrql-tutorial-control-your-data/) with more advanced concepts.
```

Reply:
Maybe use two examples to unpack what those advanced concepts are. '...with more advanced concepts, like X and Y.'

Reply:
Suggestions, "...with more advanced concepts, like rate of change and aggregation"